Source/Details/ASCollectionLayout+Subclasses.h

```diff
@@ -0,0 +1,39 @@
+//
+//  ASCollectionLayoutSubclasses.h
+//  AsyncDisplayKit
+//
+//  Created by Huy Nguyen on 21/3/17.
+//  Copyright © 2017 Facebook. All rights reserved.
+//
+
+#import <AsyncDisplayKit/ASCollectionLayout.h>
+#import <AsyncDisplayKit/ASDataController+Beta.h>
+
+@class ASCollectionContentAttributes;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ASCollectionLayout () <ASDataControllerLayoutDelegate>
+
+/// The current content, if any. This property must be accessed on main thread.
+@property (nonatomic, strong, nullable) ASCollectionContentAttributes *currentContentAttributes;
```

Comment:
Are you sure we should expose this? Users can call `-collectionViewContentSize` and `-layoutAttributesForItemAtIndexPath:` etc to get this info, if they need it for some reason.